App polish

SachinRaturi opened the pull request:
RM-41 : fixed send button in chat activity
40: addeda back button to chat activity, identify who you are talking to in the header(username).
39:remove block setting
38 : removed wallet section from setting page, added a rightmesh wallet button.
31: add MeshIM logo to header,fixed stylization
11: show animation when searching for nearby users.
cleaned up code for merge

Commit:
init commit

Commit:
RMN-41

Commit:
RMN-41,39,38,31,40

Commit:
RMN-41,39,38,31,40

Commit:
RM-41 : fixed send button in chat activity
40: addeda back button to chat activity, identify who you are talking to in the header(username).
39:remove block setting
38 : removed wallet section from setting page, added a rightmesh wallet button.
31: add MeshIM logo to header,fixed stylization

cleaned up code for merge

Commit:
RM-41 : fixed send button in chat activity
40: addeda back button to chat activity, identify who you are talking to in the header(username).
39:remove block setting
38 : removed wallet section from setting page, added a rightmesh wallet button.
31: add MeshIM logo to header,fixed stylization

cleaned up code for merge

FrazerSeymour commented on app/src/main/AndroidManifest.xml:
```diff
@@ -32,7 +32,10 @@
+        <activity android:name="io.left.meshim.activities.ChatActivity">
+        </activity>
+
+        />
```
Does this change do anything?

Commit:
RM-41 : fixed send button in chat activity
40: addeda back button to chat activity, identify who you are talking to in the header(username).
39:remove block setting
38 : removed wallet section from setting page, added a rightmesh wallet button.
31: add MeshIM logo to header,fixed stylization

cleaned up code for merge

FrazerSeymour commented on app/src/main/AndroidManifest.xml:
```diff
@@ -32,7 +30,9 @@
             android:name="io.left.meshim.activities.MainActivity"
             android:label="@string/app_name" />
-        <activity android:name="io.left.meshim.activities.ChatActivity" />
+        <activity android:name="io.left.meshim.activities.ChatActivity">
```
This whole change shouldn't be here - you have a redundant closing XML tag. Basically there shouldn't be any changes to this file.

FrazerSeymour commented:
Aside from the manifest thing everything looks good though. :+1: If you can revert that file we'll be fine to merge.

Commit:
fixed the manifest file
